e2e/uimodel/user/login.go

```diff
@@ -119,12 +73,37 @@ func (u *User) LoginWithGoogle() {
 			"should wait until google access approve button becomes active")
 
 		Expect(allowButton.Click()).To(Succeed())
-		PauseForPageJs()
+		utils.PauseForPageJs()
 	}
 }
 
+// Signout logs out a user
 func (u *User) Signout() {
 	Eventually(u.page.FindByClass("fa-sign-out"), defaults.FindTimeout).Should(BeFound())
 	Expect(u.page.FindByClass("fa-sign-out").Click()).To(Succeed())
 	Eventually(u.page.FindByClass("grv-user-login")).Should(BeFound())
 }
+
+// EnsureUser navigates to given URL and ensures that a user is logged in
+func EnsureUser(page *web.Page, URL string) {
+	log.Infof("navigating to %q", URL)
+	login := framework.TestContext.Login
+	Expect(page.Navigate(URL)).To(Succeed())
+	count, _ := page.FindByClass("grv-user-login").Count()
+
+	if count != 0 {
```

Comment:
I wonder if it's good approach to try to login behind the scenes on every step, given how ofter that function is used. 
User should be logged in in the beginning of a test, this function should be called `Navigate()` or `Go()` and another function `RequiredLoggedIn()` could assert that user is still logged in.